packages/-ember-data/tests/integration/inverse-test.js

```diff
@@ -111,41 +120,41 @@ module('integration/inverse_test - inverseFor', function(hooks) {
   });
 
   test('Returns null if inverse relationship it is manually set with a different relationship key', function(assert) {
-    Job.reopen({
+    job.reopen({
       user: belongsTo('user', { inverse: 'previousJob', async: false }),
     });
 
-    User.reopen({
+    user.reopen({
```

Comment:
We should preserve the pattern of capital letters for classes. We should be able to jsut reference the original non-registered class in these locations.